CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## VNext
 [WorkerService package is modified to depend on 2.1.1 on Microsoft.Extensions.DependencyInjection so that it can be used in .NET Core 2.1 projects without nuget errors.](https://github.com/microsoft/ApplicationInsights-dotnet/issues/1677)
-
+- [New: EventCounter to track breeze response time] (https://github.com/microsoft/ApplicationInsights-dotnet/issues/1735)
```

Comment:
rename to IngestionEndpointResponseTime

Reply:
Add link to this PR, instead of the issue. Or create a separate issue addressing this part.

Reply:
Done.